openstack/nova/values.yaml

```diff
@@ -1013,3 +1013,84 @@ vpa:
 
 # used to set the PYTHONWARNINGS environment variable everywhere
 python_warnings: "ignore:Unverified HTTPS request,ignore::SyntaxWarning"
+
+# openstack-rate-limit
+rate_limit:
+  enabled: false
+  rate_limit_by: target_project_id
+  max_sleep_time_seconds: 0
+  log_sleep_time_seconds: 10
+  backend_timeout_seconds: 1
+  whitelist:
+    - Default/service
+    - Default/admin
+    - tempest/performance-testing
+    - ccadmin/cloud_admin
+    - ccadmin/master
+  whitelist_users: null
+  blacklist: null
+  blacklist_users: null
+  groups:
+    write:
+      - delete
+      - update
+  rates:
+    default:
+      servers:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      servers/server:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
+      servers/*/migrations:
+        - action: read/list
+          limit: 100r/m
+      servers/*/migrations/migration:
+        - action: read/list
+          limit: 100r/m
+      servers/*/action:
+        - action: create
+          limit: 100r/m
+      flavors:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      flavors/flavor:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
+      os-keypairs:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      os-aggregates:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      os-aggregates/os-aggregate:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
```

Comment:
Why are we limiting these endpoints? They are admin-only, and so are in the whitelist (sic) anyway.
There are others (e.g. flavors write) -- maybe we don't need entries for admin APIs at all? Or are we protecting us from ourselves as well?